examples/source/user-consent/Advanced_User_Consent_Flow.html

```diff
@@ -85,24 +86,27 @@
 
   <!-- ## Basic usage -->
   <!--
-  The `amp-consent` component allows to specify the required user consent for this page. A CORS endpoints can be specified via the `checkConsentHref` attribute. This endpoint will be queried via a POST request to determine if the consent UI specified for it needs to be shown. The response should look like this:
+  The `amp-consent` component allows to specify the required user consent for this page using the required 'consentRequired' flag. A CORS endpoints can be specified via the `checkConsentHref` attribute along with `consentRequired` set to true. The `amp-consent` component then will check via a POST request if the consent UI needs to be shown. The response should look like this:
 
   ```json
   {
-    "promptIfUnknown": boolean (true/false)
+    "consentRequired": boolean (required),
+    "consentStateValue": enum (accepted/rejected/unknown) (optional),
+    "consentString": string (optional),
+    "expireCache": boolean (default false),
   }
   ```
 
   The response will indicate if the consent needs to be shown or not. If it needs to be shown, the element with the id given in `promptUI` will be displayed.
+ 
+  If `consentRequired` was set to true in the incline script config, then amp-consent will first check local_storage for an existing consent decision and use it if present. Otherwise it will show the promptUi (if configured). If `consentRequired` was set to false, amp-consent will immedeatly unblock all elements. 
```

Comment:
Is this the case? I think we use the localStored value if there's any without waiting for the response. Could you please verify.

Reply:
Yes that is the behavior. Do you not think that this description is conveying the message well?